src/main/java/org/sagebionetworks/web/client/widget/entity/registration/WidgetRegistrarImpl.java

```diff
@@ -326,7 +329,7 @@ public WidgetRendererPresenter getWidgetRendererForWidgetDescriptorAfterLazyLoad
     } else if (contentTypeKey.equals(REFERENCE_CONTENT_TYPE)) {
       presenter = ginInjector.getReferenceRenderer();
     } else if (contentTypeKey.equals(PROVENANCE_CONTENT_TYPE)) {
-      if (DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
+      if (featureFlagConfig.isFeatureEnabled("Provenance v2 visualization")) {
```

Comment:
Can you create an enum of feature flag keys that includes `"Provenance v2 visualization"` and use it in all of the places you have this string?

You could even change the signature of `isFeatureEnabled` so that the argument must be your feature flag key enum